website/docs/getting-started/project-walkthrough.md

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 2
+---
+
+# Project Walkthrough
+
+The template generates several files, most notably:
+
+- `./package.json` includes `scripts` to `validate`, `synth`, and `deploy` your project.
+- `./tsconfig.json` configures the `@functionless/language-service` plugin to improve IDE feedback.
+- `./cdk.json` points CDK to the application entrypoint and requires the `functionless/register` hook to transpile project files.
+- `./src/app.ts` is the appliction entrypoint.
+
+Let's take a closer look at `./src/app.ts`:
+
+```typescript
+import { App, Stack } from "aws-cdk-lib";
+import { Function, StepFunction } from "functionless";
+
+const app = new App();
+
+const stack = new Stack(app, "MyStack");
+
+const sayFunction = new Function(
+  stack,
+  "SayFunction",
+  async (event: { message: string }) => {
+    console.log(event.message);
+    return;
+  }
+);
+
+new StepFunction(stack, "Workflow", async (event: { name: string }) => {
+  await sayFunction({ message: `Hello ${event.name}` });
+});
+```
+
+We start by importing Constructs from `aws-cdk-lib` and `functionless`:
+
+```typescript
+import { App, Stack } from "aws-cdk-lib";
+import { Function, StepFunction } from "functionless";
+```
+
+:::note
+
+Most Constructs wrap some underlying Resource and provide a high-level interface for configuring the service that backs the Resource.
+
+You can read more about [Constructs][cdk-construct] and [Resources][cdk-resource] in the AWS CDK documentation.
+
+:::
+
+Functionless exposes AWS CDK compatible Resource Constructs from its top-level API.
```

Comment:
Is this redundant/confusing?

> Resource Constructs

Reply:
CDK draws a distinction between  resources and constructs. Hm maybe I could just replace Resource Constructs with Resource everywhere by reworking the intro a bit.